src/main/java/at/qcademy/tasks/oop/BuchundAutor/Bibliothek.java

```diff
@@ -0,0 +1,62 @@
+package OOP.BuchundAutor;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Bibliothek {
+
+    public List<Buch> buchListe;
+    public Buch buch;
```

Comment:
Why do you need this filed here?

Reply:
buch is an element of the buch aray list, which i can remove or add

Reply:
You don't need this field here, it is not used anywhere